spinta/manifests/internal_sql/commands/auth.py

```diff
@@ -74,11 +73,14 @@ def internal_authorized(
     # Protected and higher level nodes can be accessed with parent nodes scopes.
     if access > Access.private:
         scopes.extend(parents)
-
     if not isinstance(action, (list, tuple)):
         action = [action]
-    scopes = [internal_scope_formatter(context, scope, act) for act in action for scope in scopes]
-
+    scopes = [
+        internal_scope_formatter(context, scope, act, udts)
+        for act in action
+        for scope in scopes
+        for udts in [False, True]
```

Comment:
```suggestion
        for is_udts in [False, True]
```

Reply:
> Is the recommendation only for this place, or to change udts -> is_udts everywhere?

I would say - for all places, where it is `boolean`, i.e. where we could answer to the question "is udts" with one of two options - yes (True) or no (False).

Reply:
Yeah, like the comment above, we should get used to using `is_` prefix for booleans, helps massively w/ readability